### ISSUE TITLE
C# debugging fails in v1.54 of VS Code

### ISSUE THREAD

**Issue by howardginsburg** [microsoft/vscode-azurefunctions#2656]
With the most recent update to the Insiders version, I cannot debug an Azure Functions project.

Steps to reproduce:
1. Run -> Start Debugging
2. The project will build and the Azure Function runtime will start.
3. Error window in right corner of VS Code says "Cannot read property 'map' of undefined".
4. Debug stops.
5. Azure Function runtime must be killed through task manager or my shutting down VS Code.

**Comment by ejizba** [comment]
What type of project is this - C#, JavaScript, etc.? And what version of the func cli do you have?

**Comment by xsoheilalizadeh** [comment]
@ejizba 

I have the same issue with the latest version v1.2.1, the project is c#.

```
                  %%%%%%
                 %%%%%%
            @   %%%%%%    @
          @@   %%%%%%      @@
       @@@    %%%%%%%%%%%    @@@
     @@      %%%%%%%%%%        @@
       @@         %%%%       @@
         @@      %%%       @@
           @@    %%      @@
                %%
                %


Azure Functions Core Tools
Core Tools Version:       3.0.3233 Commit hash: d1772f733802122a326fa696dd4c086292ec0171
Function Runtime Version: 3.0.15193.0

Version: 1.54.0-insider (user setup)
Commit: cd97629c4458c05b16a1fd444d21c61295ba2529
Date: 2021-02-09T08:37:01.449Z
Electron: 11.2.3
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
```

**Comment by howardginsburg** [comment]
C# project
CLI: 3.0.3233

**Comment by ejizba** [comment]
Hey are you guys still seeing this behavior? I saw some problems for a few days but it kind of went away - maybe VS Code fixed the bug on their end...

**Comment by howardginsburg** [comment]
Yes, the problem still exists for me even after updating to the latest VSCode Insider build and Functions extension.

**Comment by ejizba** [comment]
What OS are you on? I still haven't seen the particular error "Cannot read property 'map' of undefined"...

**Comment by howardginsburg** [comment]
Edition	Windows 10 Enterprise
Version	20H2
Installed on	‎10/‎13/‎2020
OS build	19042.804
Serial number	037373502857
Experience	Windows Feature Experience Pack 120.2212.551.0

**Comment by runtimeware** [comment]
Same issue here on Windows 10 on a C# project: 
 
Azure Functions Core 3.0.3331
Extension v: 1.2.1

**Comment by ejizba** [comment]
I've identified the root cause and filed this issue on VS Code: https://github.com/microsoft/vscode/issues/118256

As a workaround, I can get this working off-and-on by using the "trash can" icon to stop the task and restarting debugging.
<img width="340" alt="Screen Shot 2021-03-05 at 12 38 52 PM" src="https://user-images.githubusercontent.com/11282622/110171140-23024700-7db0-11eb-8f1f-521993f4a0a4.png">
You can also temporarily downgrade your version of VS Code to v1.53.

Also fwiw, this reproduces on mac/linux with a different error "No process with the specified id is currently running."
<img width="267" alt="Screen Shot 2021-03-05 at 12 43 48 PM" src="https://user-images.githubusercontent.com/11282622/110171357-74123b00-7db0-11eb-9ef1-bdb0b63cb943.png">

**Comment by ejizba** [comment]
Hey all - we included a workaround for this bug in the v1.3.0 release last week. I'm still hoping to get the root-cause VS Code bug fixed, but if you update our extension it should be working now.

**Comment by ejizba** [comment]
The root cause bug in VS Code was fixed a while ago for this.
